Interfaces/API/Job.py

```diff
@@ -238,9 +242,10 @@ def setInputSandbox( self, files ):
 
   #############################################################################
   def setParametricInputSandbox( self, files ):
-    """Helper function.
+    """Helper function. Obsoleted, for backward compatibility only !
```

Comment:
Use the deprecated markup
http://www.sphinx-doc.org/en/stable/markup/para.html#directive-deprecated